python/ray/tests/test_placement_group.py

```diff
@@ -506,6 +522,51 @@ def f():
             ray.get(task_ref)
 
 
+def test_remove_placement_group_worker_startup_slowly(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=4)
+    ray.init(address=cluster.address)
+
+    placement_group = ray.util.placement_group([{"CPU": 2}, {"CPU": 2}])
+    assert placement_group.wait(10)
+
+    @ray.remote(num_cpus=2)
+    class A:
+        def f(self):
+            return 3
+
+    @ray.remote(num_cpus=2, max_retries=0)
+    def long_running_task():
+        print(os.getpid())
+        import time
+        time.sleep(60)
+
+    # Schedule a long-running task that uses
+    # runtime env to mock worker start up slowly.
+    task_ref = long_running_task.options(
+        placement_group=placement_group,
+        runtime_env={
+            "plugins": {
+                MOCK_WORKER_STARTUP_SLOWLY_PLUGIN_CLASS_PATH: {}
+            }
+        }).remote()
+    a = A.options(placement_group=placement_group).remote()
+    assert ray.get(a.f.remote()) == 3
+
+    ray.util.remove_placement_group(placement_group)
+
+    # Make sure the actor has been killed
+    # because of the removal of the pg.
+    # TODO(@clay4444): Make it throw a `ActorPlacementGroupRemoved`.
```

Comment:
Is this not done in this PR? Should we touch GCS actor manager?

Reply:
Yeah, it's not done in this PR and I will do it in the next PRs.